cypress/tests/integration/Plugins.spec.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file cypress/tests/integration/Plugins.spec.js
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ */
+
+describe('Plugins tests', function() {
+    it('Check if Plugins.xml file was cached', function() {
+        cy.server();
+        cy.route('GET', '**/grid/plugins/plugin-gallery-grid/fetch-grid*').as('getPluginList');
+        cy.login('dbarnes', null, 'publicknowledge');
+
+        cy.get('a:contains("Website")').click();
+        cy.waitJQuery();
+        cy.get('button#plugins-button').click();
+        cy.get('button#pluginGallery-button').click();
+        cy.wait('@getPluginList', {
+            responseTimeout: 30000
```

Comment:
This `responseTimeout` acts a trigger to avoid a too long response time on `plugin-gallery-grid/fetch-grid` response.